src/utils/Xhr.js

```diff
@@ -77,6 +84,7 @@ class Xhr {
         sharedLinkPassword,
         responseInterceptor,
         requestInterceptor,
+        retryableStatusCodes = [HTTP_STATUS_CODE_RATE_LIMIT],
```

Comment:
If `options` is defined, but this property isn't, we lose the default value here. Should we put it inline on line 100?

Reply:
Are you sure?

```
> function foo({a = 'A!', b} = {}) { return {a, b} }
undefined
> foo()
{ a: 'A!', b: undefined }
> foo({b: 'z'})
{ a: 'A!', b: 'z' }
```

Reply:
Hm, fair enough. Brain fart, I guess.